### DISCUSSION TITLE
Effects (like blur) and color adjustments

### DISCUSSION THREAD

**Discussion by Pigeonwingattack** [keiwando/toonsquid#459] [Ideas]
First of all, I’m so happy to find toonsquid and I’m beyond amazed at how it works. I’m loving it so far and can’t stop saying amazing things about it!

I hope in a future update we can get effects, or adjustment layers. Stuff like curves, hue and saturation, blur, etc. i guess for now that has to be done in other softwares and I appreciate that toonsquid so far has been adding updates that are great QOL. I would indeed trade the effects for the QOL updates it has, any time. But with the app getting better and better in the animation side of things, the next step having the support for some effects like blur and color adjustments would be amazing.

Great work, and can’t wait to see what I can fully do with this app!

**Comment by jsllllc** [comment]
Blur and Hue/Saturation would be incredibly useful for creating shadows and glow effects. I currently export from TS to Rough Animator and use its blur-edged fill tool to manually create shadow layers frame-by-frame (tedious, but works). I had bought ProCreate Dreams thinking it could take care of this step but refunded when I saw how poorly it works with other apps.

**Comment by Lioncub901** [comment]
Yes, blur and hue/saturation. Also color balance and tone curves, brightness contrast and more. Just basically a vfx node editor would be cool.
